test/uk/gov/hmrc/agentsubscription/controllers/SubscriptionJourneyControllerSpec.scala

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscription.controllers
+
+import org.mockito.ArgumentMatchers.{ any, eq => eqs }
+import org.mockito.Mockito._
```

Comment:
should try to use scalamock over mockito when writing new tests, this was a team decision. Could have a discussion about our testing practices.